portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,93 @@
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+    margin-left: auto;
+    margin-right: auto;
+    width: 650px;
+    font-family: sans-serif;
+    display: block;
+    text-align: center;
+    padding: 10px;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+/*make portrait image round*/
+img.rounded{
+        display: block;
+        margin-left: auto;
+        margin-right: auto;
+        border-radius: 50%;
+        width: 300px;
+        height: auto;
+        margin-top: 35px;
+}
+
+/*make artist images round*/
+img.rounded-corners {
+    border-radius: 50%;
+    padding: 70px;
+}
+
+/*create box for about me class*/
+.aboutme {
+    
+    background-color: #c9d6df;
+    height: 1175px;
+    margin-right: auto;
+    margin-left: auto;
+    font-family: sans-serif;
+    display: block;
+    text-align: center;
+    padding: 10px;
+}
+
+h1 {
+    padding: 10px;
+    font-size: 50px;
+    text-align: center;
+}
+h2 {
+    padding: 10px;
+    font-size: 30px;
+    text-align: center;
+}
+
+h3{
+    padding: 10px;
+    font-size: 30px;
+    text-align: left;
+}
+
+/*rules for paragraphs*/
+p{
+    margin-left: auto;
+    margin-right: auto;
+    width: 650px;
+    font-family: sans-serif;
+    display: block;
+    text-align: center;
+    padding: 10px;
+}
+
+/* create rules for paragraph in blog*/
+p1{
```

Comment:
Only `<p>` tag exists for HTML, `<p1>` will be a custom tag defined by you. Personally, I would use the `<p>` tag for the blog post text, and create a separate class, e.g. `.blog-text`. class to specify the properties you want for blog post. Since a class has higher priority than elements like `<p>`, even if you specify a field that applies for all `<p>`, having the same field in the class will overwrite it.

https://www.w3schools.com/css/css_specificity.asp